README.md

```diff
@@ -513,6 +513,113 @@ export const getConfig = async () => {
 };
 ```
 
+### Slices
```

Comment:
I think we should explain more about the use cases in the Guides.
For README, we keep it minimal, and it looks good at this point.